niceday-api/api/openapi.yaml

```diff
@@ -106,6 +114,26 @@ paths:
                 items:
                   $ref: '#/components/schemas/SmokingTrackerEntry'
       x-swagger-router-controller: UserTrackers
+  /acceptconnection:
+    post:
+      summary: Accept the connection request specified in the body
+      operationId: setAcceptConnection
+      requestBody:
+        content:
+          application/json:
+            schema:
+              required:
+                - invitation_id
+              type: object
+              properties:
+                invitation_id:
+                  type: string
+                  description: ID of the connection request
+                  example: "38527"
+      responses:
+        "200":
```

Comment:
Can this only ever be successful?

Reply:
Looks like 500 is also a possible response (also for the connectionrequests endpoint). Not sure if it really matters whether they are in the openapi yaml or not.

Reply:
We still get an error response from the service, but I agree that it is not the cleanest way